websites/C/Coub/presence.ts

```diff
@@ -322,26 +308,33 @@ presence.on("UpdateData", async () => {
 	const query: { [key: string]: unknown } = getQuery(),
 		context = pages.find(x => x.middleware(window, [query]));
 	if (!context) return;
+	const data: PresenceData = {
+			largeImageKey: "logo",
+		},
+		showStartedBrowsing = await presence
+			.getSetting<boolean>("show_startedBrowsing")
+			.catch(() => true);
```

Comment:
Why is the catch used here?

Reply:
to return the default value,
getSetting can reject if its undefined
https://github.com/PreMiD/Extension/blob/5e9894de2c5e82d9425123ab353859158413e65d/src/util/presence/devHelper.ts#L411

Reply:
My apologies, but I see what you mean. Myself and Bas didn't realise this 😆